### PR TITLE
Migrate CI script to GitLab

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: None

Signed-off-by: Fabio Tranchitella <fabio@tranchitella.eu>

### CODE REVIEW

**Comment by tranchitella** [comment]
recheck

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -16,12 +16,9 @@ before_script:
...
     # Get last remaining tags, if any.
     - git fetch --tags origin
+    # Set up git identity
+    - git config --global user.name "Northern.tech" && git config --global user.email "info@northern.tech"
 
-    - git clone git://github.com/mendersoftware/mendertesting
-
-script:
-    # Check commit compliance.
-    - mendertesting/check_commits.sh
```
Comment (lluiscampos) [inline .gitlab-ci.yml]: This check we shouldn't remove.

The modern way of having it is just importing it from `mendertesting`. See [here](https://github.com/mendersoftware/mender-convert/blob/next/.gitlab-ci.yml#L7-L8) for an example.

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -16,12 +16,9 @@ before_script:
     - git fetch origin 'refs/heads/*:refs/heads/*'
     # Get last remaining tags, if any.
     - git fetch --tags origin
```
Comment (lluiscampos) [inline .gitlab-ci.yml]: I don't think this is necessary anymore. It was there for the check_commits which has been removed

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-    depth: false
 
-env:
-    global:
-        TEST_BRANCH: $TRAVIS_BRANCH
+stages:
+  - test
 
-before_script:
+test:unit:
```
Comment (lluiscampos) [inline .gitlab-ci.yml]: There are no unit tests as such here, I suggest we rename the job to `test:changelog-generator` or something generic like `test:extra-tools`

Comment (tranchitella) [reply .gitlab-ci.yml]: Done, using `test:extra-tools`

File: .gitlab-ci.yml (hunk 2 above)
Comment (tranchitella) [reply .gitlab-ci.yml]: Removed.

**Pull request merged**